backend/bin/server/cmd/profile.go

```diff
@@ -34,6 +34,5 @@ func getBaseProfile(dataDir string) *config.Profile {
 		LastActiveTs:       time.Now().Unix(),
 		Lsp:                flags.lsp,
 		PreUpdateBackup:    flags.preUpdateBackup,
```

Comment:
Can you help with deleting flags.preUpdateBackup as well?

Reply:
ok